helpful_to_new_devs.md

```diff
@@ -0,0 +1,14 @@
+## First pull
+ - May require a bundle install before guard will run
+
+## Mono
+ - Mono is required by guard, it will also be required if not already installed
```

Comment:
FYI - If you're using Homebrew (and you hopefully are!) then `brew install docfx` will install Mono as a dependency.